saskatoon/harvest/signals.py

```diff
@@ -37,6 +38,30 @@ def changed_by(sender, instance, **kwargs):
     else:
         instance.changed_by = None
 
+def notify_unselected_pickers(sender, instance, **kwargs):
+    if instance.id:
+        original_instance = sender.objects.get(id=instance.id)
+        if instance.status == "Ready" and original_instance.status != "Ready":
+            unselected_pickers = instance.get_unselected_pickers()
```

Comment:
So people that cancelled their participation will still receive an email ?

Reply:
No, pickers that cancelled will not be notified. Only pickers that got rejected or those which are still pending will receive the emails.